app/views/_user_util_links.html.erb

```diff
@@ -1,27 +1,26 @@
 <ul id="user_utility_links" class="navbar-nav">
-  <%= render 'shared/locale_picker' if available_translations.size > 1 %>
+  <!-- <%= render 'shared/locale_picker' if available_translations.size > 1 %>-->
   <% if user_signed_in? %>
     <li class="nav-item">
       <%= render_notifications(user: current_user) %>
     </li>
     <li class="nav-item dropdown">
-      <%= link_to hyrax.dashboard_profile_path(current_user), role: 'button', class: 'nav-link dropdown-toggle', data: { toggle: 'dropdown' }, aria: { haspopup: true, expanded: false, controls: 'user-util-links' } do %>
+      <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
Can you share the reason behind this change?

Reply:
Sure.  I think the `a.dropdown-toggle` element (from looking at Bootstrap 4 documentation), might serve as a toggle element and not necessarily a navigation element.  Clicking it opens/closes the dropdown menu.  If it's navigable, then clicking on it would take the user to the link instead of triggering the dropdown UI state.

https://getbootstrap.com/docs/4.0/components/navbar/

Reply:
My only question with changes like this is whether it is conceivable a user might want to right-click here and click “open in new tab” instead of toggling the state. If that’s not very likely (for example because there’s a better / more obvious way of getting to that page) then I’m fine with it.

Reply:
Ah, yea however you want to do it is fine with me.  Doesn't look like the v4 documentation provides any examples of the toggle being navigable, and it maybe gives an example to follow Bootstrap implementation as closely as possible?

Reply:
Considering that things like command-click and middle-click already don’t seem to work to open this link in a new tab, I think replacing this with `#` is probably fine. It might be nice to have a “profile” link in this dropdown, but I’m not sure translations exist for it and don’t really want to worry about that right now 😅.